qcodes/dataset/data_set.py

```diff
@@ -405,8 +405,6 @@ def add_results(self, results: List[Dict[str, VALUE]]) -> int:
         len_before_add = length(self.conn, self.table_name)
         insert_many_values(self.conn, self.table_name, list(expected_keys),
                            values)
-        # TODO: should this not be made atomic?
-        self.conn.commit()
```

Comment:
So this is technically an api change for anyone that uses add_results directly. Do we need to worry about that? Should we rather add a new function that does not commit and make this an atomic around that?

Reply:
`insert_many_values` already used to commit, so this line is just redundant. There is not API change at all, neither for `add_results`, nor for `insert_many_values`